src/Compiler/Compiler/2_ConvertingXamlToCSharp/GeneratingCSharpCode.cs

```diff
@@ -1407,7 +1407,7 @@ static string GenerateStoryboardAccessToPropertyBlockCode(string namespaceName,
                 {
                     //todo: this
                     string index = lastPartOfBlock.TrimEnd(']').TrimStart('[');
-                    generatedCode = string.Format("var {0} = ((dynamic){1})[{2}];", currentVarName, blockRootElement, index);
+                    generatedCode = string.Format("var {0} = ((System.Collections.IList){1})[{2}];", currentVarName, blockRootElement, index);
```

Comment:
For consistency the rest of the generated code
```suggestion
                    generatedCode = string.Format("var {0} = ((global::System.Collections.IList){1})[{2}];", currentVarName, blockRootElement, index);
```